.semaphore/semaphore.yml

```diff
@@ -16,10 +16,12 @@ blocks:
         - name: 'FV (DevStack + Tempest)'
           commands:
             - checkout
-            - ln -s . networking-calico
+            - git checkout -b devstack-test
+            - mkdir -p ~/calico
+            - ln -s `pwd` ~/calico/networking-calico
```

Comment:
The 3 lines above allow the following devstack/bootstrap.sh script to use this existing networking-calico checkout instead of doing a fresh `git clone https://github.com/projectcalico/networking-calico` - which would check out master and so not the code we actually want to test.  I thought I already had that with the `ln -s . networking-calico`, but that was broken because
1. `.` needs to be `$(pwd)`
2. the link needs to be in a subdir
3. we need to have a branch name for HEAD.